assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -385,6 +385,21 @@ const SearchFunnelWidget = ( {
 					</Grid>
 				) }
 
+			{ ! isAnalyticsActive &&
+				isAnalyticsConnected &&
```

Comment:
It's not possible for a module to be inactive and connected so this condition will never render.

Instead of adding a mostly duplicate block here, let's update the above to render if `! isAnalyticsActive || ! isAnalyticsConnected` (could really be simplified to the latter but perhaps easier to understand this way) rather than only for the inactive case. Then the `incomplete` prop would be set as above `active && ! connected` to cover both cases.

Reply:
Beautiful!